database/sqlite/sqlite_aclk.c

```diff
@@ -303,6 +366,13 @@ void sql_aclk_sync_init(void)
     info("SQLite aclk sync initialization completed");
     fatal_assert(0 == uv_mutex_init(&aclk_async_lock));
 
+    if (likely(rrdcontext_enabled == CONFIG_BOOLEAN_YES)) {
```

Comment:
Considering that this PR is going to be merged soon, I ran `converity-scan` on it, and I got the following reports:

```sh
*** CID 355302:  Code maintainability issues  (UNUSED_VALUE)
/database/sqlite/sqlite_aclk.c: 370 in sql_aclk_sync_init()
364             }
365         }
366         info("SQLite aclk sync initialization completed");
367         fatal_assert(0 == uv_mutex_init(&aclk_async_lock));
368     
369         if (likely(rrdcontext_enabled == CONFIG_BOOLEAN_YES)) {
>>>     CID 355302:  Code maintainability issues  (UNUSED_VALUE)
>>>     Assigning value from "sqlite3_exec(db_meta, "SELECT host_id, hostname, registry_hostname, update_every, os, timezone, tags, hops, memory_mode, abbrev_timezone, utc_offset, program_name, program_version, entries, health_enabled FROM host WHERE hops >0;", create_host_callback, NULL, NULL)" to "rc" here, but that stored value is overwritten before it can be used.
370             rc = sqlite3_exec(db_meta, "SELECT host_id, hostname, registry_hostname, update_every, os, "
371                "timezone, tags, hops, memory_mode, abbrev_timezone, utc_offset, program_name, "
372                "program_version, entries, health_enabled FROM host WHERE hops >0;",
373                   create_host_callback, NULL, NULL);
374         }
375     

** CID 355301:  Error handling issues  (CHECKED_RETURN)
/database/rrdcontext.c: 1831 in rrdhost_find_by_node_id()
````

Reply:
Fixing..